src/mbpfan.c

```diff
@@ -135,7 +135,7 @@ t_sensors *retrieve_sensors()
 
     char *path = NULL;
     char *path_begin = NULL;
-
+    
```

Comment:
Whitespace change.

Reply:
Can you give me a hint which linter to use for auto formating? Or do you have a link to emacs/eclipse settings?